src/bindings/js/node/src/core_wrap.cpp

```diff
@@ -232,6 +233,40 @@ Napi::Value CoreWrap::get_available_devices(const Napi::CallbackInfo& info) {
     return js_devices;
 }
 
+Napi::Value CoreWrap::get_versions(const Napi::CallbackInfo& info) {
+    // If no arguments are passed into the method.
+    if(info.Length()==0) {
```

Comment:
Please, make sure that your code formatted using openvino [clang format](https://github.com/openvinotoolkit/openvino/blob/master/src/.clang-format).